### DISCUSSION TITLE
EP Crashes if Computer is Running Close to 100% CPU

### DISCUSSION THREAD

**Discussion by WillMartin58** [valinet/ExplorerPatcher#2001] [Q&A]
I'm finding that the latest version of EP will crash repeatedly if my computer is running at close to 100% CPU for a very long time, and due to the amount of background software it's loading, this means each morning until my computer calms down (15 or 20 mins), I can't reliably open a file explorer window.  Has anyone else noticed this?  Removing and reloading EP will fix this sometimes, but not always.

**Comment by pyrates999** [comment]
Why is the cpu being pegged to 100%?

**Comment by WillMartin58** [comment]
It's pegged because the computer is starting up.  It normally runs under 20%, but on startup, it'll stay at 100% for a few minutes.

**Comment by winklejude** [comment]
me either
